kimchi/src/circuits/polynomials/lookup.rs

```diff
@@ -261,6 +261,9 @@ pub struct LookupConfiguration<F: FftField> {
     /// particular row, so that we can treat each row uniformly as having the same number of
     /// lookups.
     pub dummy_lookup_value: Vec<F>,
+
+    /// The table ID to combine with the placeholder lookup value.
+    pub dummy_lookup_table_id: i32,
```

Comment:
why not store as `F` directly?

Reply:
I have an issue for combining this with the table value as a `JointLookup`. This PR predates the API change, so it was easier at the time to use an `i32`; I'll update this after merging.